src/Interceptor/Impl/ListenInterceptor.php

```diff
@@ -19,6 +20,9 @@ public function startUp(Instance $instance): Response
             ->getAttributesInstances(Listen::class);
 
         foreach ($attributes as $attribute) {
+            if ($this->isInfinityLoop($attribute->name, $instance->getReflection()->getName())) {
+                continue;
```

Comment:
No exception in order to prevent breaking the process, so yes it will be just ignored silently.